apps/plato_single_variant/Readme.md

```diff
@@ -83,27 +86,24 @@ User don't need to specify `load-data` or `load-trait` commands here since app s
 **Case Threshold**: It will exculude the assocaition results if they do not meet the threshold of number cases provided in this option. It is only applicable to case-control PheWAS analysis.
 
 ## Common
-
 **Ouput Filename**: The filename of the output of the regression. If not provided, default file name `output.txt` will be used.
```

Comment:
**Output Filename**: The filename of the output of the regression. If not provided, the default file name `output.txt` will be used.